### DISCUSSION TITLE
Double-click to select multiple word?

### DISCUSSION THREAD

**Discussion by MedievalSteel** [sumatrapdfreader/sumatrapdf#4208] [Q&A]
Is there any way?
Like firefox or pdf.js

**Comment by GitHubRulesOK** [comment]
Native PDF is not suited to the task. Both the two main examples are naturally slower than seek a letter then another letter etc.

The advantage Mozilla have leveraged is they extract and write all the text from the pdf into a separate set of pages (text overlay on images) So you are looking at duplicated HTML search not the real PDF. 

The Advantage Adobe Acrobat has is they extract all the text for audio accessibility so they can index the duplicated accessible text.

Both systems consume much more resources (slowdown to a degree, maintaining much more data, but faster for search) and have similar limited application usage when fonts go wrong. However it does mean they can search in areas SumatraPDF cannot such as annotation!

For searching multiple files or associated data such as annotations/bookmarks in an assisted reader use Acrobat
For Searching faster in HTML multiple document hits use Firefox.
For fast searching  literal letter sequence in one PDF body text (not annotations or bookmarks) use SumatraPDF

**Comment by kjk** [comment]
It's not possible currently. It's a reasonable suggestions so feel free to open a feature request via https://github.com/sumatrapdfreader/sumatrapdf/issues/new?assignees=&labels=&projects=&template=feature_request.md&title= so that it's tracked
